Tomlet/Extensions/ReflectionExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+
+namespace Tomlet.Extensions
+{
+    public static class ReflectionExtensions
```

Comment:
Please make extension classes internal so they don't get exposed to consumers of the library